### PR TITLE
Document the skipExcluded option of the Analyzer

### PR DESCRIPTION
**Pull request by oheger-bosch**
Refer to the single commits for further information.

### CODE REVIEW

File: docs/config-file-ort-yml.md [hunk 1]
```diff
@@ -27,19 +27,34 @@ Exclusions apply to paths (files/directories) or scopes. Examples of currently s
...
-dependencies.
+ORT's default philosophy is to analyze and scan everything it can find to build a complete picture of a repository and
+its dependencies.
 
 However, the users may not be interested in the results for components that are not included in their released
 artifacts, for example build files, documentation, examples or test code. To support such use cases, ORT provides a
 mechanism to mark files, directories or scopes included in the repository as excluded.
 
-Note that the excluded parts are analyzed and scanned, but are treated differently in the reports ORT generates:
+Note that in the standard mode the excluded parts are analyzed and scanned, but are treated differently in the reports
```
Comment (sschuberth) [inline docs/config-file-ort-yml.md:37]: Proposal: s/in the standard mode/by default/ to match "default" from line 30.

File: docs/config-file-ort-yml.md [hunk 2]
```diff
@@ -27,19 +27,34 @@ Exclusions apply to paths (files/directories) or scopes. Examples of currently s
...
+ORT generates:
 
 * The issue summary does not show issues in the excluded parts.
 * The excluded parts are grayed out.
 * The reason for the exclusion is shown next to the result.
 
+This is a rather safe option, since the reports still display elements marked as excluded and thus allow the user to
```
Comment (sschuberth) [inline docs/config-file-ort-yml.md:44]: "allow*s*"

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: docs/config-file-ort-yml.md (hunk 2 above)
Comment (oheger-bosch) [reply docs/config-file-ort-yml.md:44]: Not sure about this one: The "allow" refers to "reports" as subject.

**Comment by oheger-bosch** [comment]
I think the build failure should disappear after #6452 was merged.

File: docs/config-file-ort-yml.md (hunk 2 above)
Comment (sschuberth) [reply docs/config-file-ort-yml.md:44]: You're right.

**Comment by sschuberth** [comment]
> I think the build failure should disappear after #6452 was merged.

I've merged it, please rebase.

**Review by sschuberth — DISMISSED** [review]

**Review by sschuberth — APPROVED** [review]

**Pull request merged**